qemu/tests/nic_hotplug.py

```diff
@@ -115,9 +115,10 @@ def device_add_nic(pci_model, netdev, device_id):
             root_port_id = bus.get_free_root_port()
             if root_port_id:
                 pci_add_cmd += ",bus=%s" % root_port_id
-                root_port = vm.devices.get_buses({"aobject": root_port_id})[0]
-                root_port.insert(qdevices.QBaseDevice(pci_model,
-                                                      aobject=device_id))
+                if used_sameid != "yes":
```

Comment:
Hello @yanglei-rh, for negative test of netdev, we don't need to assign bus and addr to it. Can you run "nic_hotplug.used_netdev.nic_virtio.default" to check it can work now?

Reply:
Hi @PaulYuuu, it's works well now.

Reply:
Okay, thanks.